src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityProcessingArray.java

```diff
@@ -252,6 +252,11 @@ public void findMachineStack() {
             this.currentMachineStack = machine;
         }
 
+        @Override
+        protected int getOverclockForTier(long voltage) {
+            return super.getOverclockForTier(machineVoltage);
```

Comment:
This should be the minimum of the machine voltage and the voltage that the PA is running at